BeagleBone_black/src/data/data.hpp

```diff
@@ -129,33 +136,23 @@ struct Batteries {
 // Motor data
 // -------------------------------------------------------------------------------------------------
 
-enum MotorState {
-  kCriticalFailure,
-  kPreOperational,
-  kMotorAccelerating,
-  kMotorDecelerating,
-  kMotorStopping,
-  kMotorStopped
-};
-
-struct Motors {
-  MotorState current_motor_state;
-  int32_t motor_velocity_1;
-  int32_t motor_velocity_2;
-  int32_t motor_velocity_3;
-  int32_t motor_velocity_4;
-  int16_t motor_torque_1;
-  int16_t motor_torque_2;
-  int16_t motor_torque_3;
-  int16_t motor_torque_4;
+struct Motors : public Module {  // @TODO (Sean) Uncomment and integrate
```

Comment:
Is the `TODO` still relevant? @Seanmullan

Reply:
No, thanks for spotting that